aws_lambda_builders/__main__.py

```diff
@@ -8,37 +8,76 @@
 
 import sys
 import json
+import os
+import logging
+
 from aws_lambda_builders.builder import LambdaBuilder
 from aws_lambda_builders.exceptions import WorkflowNotFoundError, WorkflowUnknownError, WorkflowFailedError
 
 
-def _success_response(request_id):
+log_level = int(os.environ.get("LAMBDA_BUILDERS_LOG_LEVEL", logging.INFO))
+
+# Write output to stderr because stdout is used for command response
+logging.basicConfig(stream=sys.stderr,
+                    level=log_level,
+                    format='[Lambda Builders] %(asctime)s - %(levelname)s - %(message)s')
+
+LOG = logging.getLogger(__name__)
+
+
+def _success_response(request_id, artifacts_dir):
     return json.dumps({
         "jsonrpc": "2.0",
         "id": request_id,
-        "result": {}
+        "result": {
+            "artifacts_dir": artifacts_dir
+        }
     })
 
 
-def main():
+def _error_response(request_id, http_status_code, message):
+    return json.dumps({
+        "jsonrpc": "2.0",
+        "id": request_id,
+        "error": {
+            "code": http_status_code,
+            "message": message
+        }
+    })
+
+
+def main():  # pylint: disable=too-many-statements
     """
     Implementation of CLI Interface. Handles only one JSON-RPC method at a time and responds with data
+
+    Input is passed as JSON string either through stdin or as the first argument to the command. Output is always
+    printed to stdout.
     """
 
     # For now the request is not validated
-    request = json.load(sys.stdin)
+    if len(sys.argv) > 1:
+        request_str = sys.argv[1]
```

Comment:
Does that mean its anything passed as an argument right now?

Reply:
lol yes. I am trying to incrementally evolve the CLI interface. I know this is bad, but before release, we will get a more stabler CLI interface